distance.go

```diff
@@ -7,77 +7,46 @@ package units
  */
 
 import (
+	"fmt"
 	"math"
 	"strconv"
 	"strings"
 )
 
-// Distance represents a quantity of length.
-// Internally, it stores the distance in metres, as a float64.
-// Create a new Distance by multiplying one of the constants:
-// d1 := 100 * units.Metre
-// d2 := 10 * units.NauticalMile
-// Use methods like Metres() or Kilometres() to get the distance in the unit of your choice.
-// US and UK spelling of consts and methods supported.
-
-type Distance float64
+// Distance represents a quantity of speed.
+type Distance interface {
```

Comment:
need 
Short() string
Name() string
in the interface for units.Distance.Short() to work.

Reply:
Ah... it's in `unit.go`, which I forgot to add, added now.